src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -195,6 +203,8 @@
 
     private double overallHappiness = AVERAGE_HAPPINESS;
 
+    private List<EntityLivingBase> guardTargets;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "guardTargets" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)